src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    $CommonMetadataElements$
```

Comment:
How does this variable get populated?

Reply:
[Arcade targets](https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Workarounds.targets#L69) take care of that. 
They insert the same values based on msbuild properties as standard NuGet Pack target would.